api-reference/node.mdx

```diff
@@ -34,8 +34,8 @@ yarn add @getmetal/metal-sdk
 | Param       | Type   | Description                                                                     |
 | ----------- | ------ | ------------------------------------------------------------------------------- |
 | `api_key`   | string | The API Key for your Org. [Learn more.](/misc-get-keys)                         |
```

Comment:
if we're gonna update to camelCase, let's be consistent for `api_key` to `apiKey` as well

Reply:
ah good catch! updated